testinfra/conftest.py

```diff
@@ -26,19 +25,18 @@ def securedrop_import_testinfra_vars(hostname, with_header=False):
     Vars must be stored in `testinfra/vars/<hostname>.yml`.
     """
     filepath = os.path.join(os.path.dirname(__file__), "vars", hostname+".yml")
-    file_load = open(filepath, 'r')
+    with open(filepath, 'r') as f:
+        hostvars = yaml.safe_load(f)
```

Comment:
I noticed this in `securedrop-admin`:

```
106:            site_config_vars = yaml.safe_load(site_config_file.read())
```

so tested and confirmed that `pyyaml` will take either a `file` or `str` object.